### ISSUE TITLE
How can i send bulk message with image?

### ISSUE THREAD

**Issue by AsheKR** [xtrinch/fcm-django#158]
I am currently using it as below.

```
image_url = 'imaeg_url'  # Where should I go in?
queryset.send_message(title='Team Push', body='Team Push Test Content', data={'PUSH_CODE': 'TEAM_PUSH'})
```

How can I send image_url in `queryset.send_message`?

**Comment by Andrew-Chen-Wang** [comment]
When we finish the migration from v1, you can do:

```python
from firebase_admin.messaging import Message, Notification

queryset.send_message(Message(notification=Notification(title="", body="", image="")))
```

**Comment by xtrinch** [comment]
Released with 1.0.0. Feel free to re-open if any questions remain.
